pyV2DL3/EventClass.py

```diff
@@ -0,0 +1,64 @@
+import ROOT
```

Comment:
This is a VEGAS specific module, which requires access to data products like effective areas which are only available in this format for VEGAS. My suggestion would be to move this module into the pyV2Dl3/vegas directory. This way we have clearly separated modules required by both chains (in ./pyV2Dl3) and those required for one or the other.